source/index.html.md

```diff
@@ -77,10 +77,10 @@ Number  Text
 
 ## Signing requests
 
-Requests can optionally be signed.
+POST requests must be signed.
```

Comment:
Shouldn't you also state that other requests *can* be signed? If I didn't knew otherwise, I would have a brief second thought before signing every request.

Reply:
I think the only non-POST request is `GET /account`. This request has an empty body, so one would always be signing the empty string. The gateway accepts a `Signature` header in the `GET /account` request, however, it is ignored.

Am I missing something?

Reply:
I think that's good to know. I simply didn't special-case neither POST nor empty bodies (I remember asking back then to be sure), so I'm signing regardless, I think that is the nicest as a consumer.

Reply:
I have pushed https://github.com/clearhaus/gateway-api-docs/pull/92/commits/b20346aa7f0a5dd3aa4887224d9fd8ad485b10f0.